docs/workflows/data/google-big-query.md

```diff
@@ -0,0 +1,48 @@
+# GCP BigQuery
+
+The GCP (Google Cloud Platform) BigQuery node allows a workflow to interact with Google BigQuery by fetching, creating, updating, or deleting data and optionally store the result on the workflow payload.
```

Comment:
store => storing (to match tense)

Reply:
I'd also say "the result of the operation"

Reply:
Can you add a link to "Google BigQuery" going to one of their pages with a general overview of what the service is? We do that for other nodes that interact with third-party services.